src/client/src/.tests/components/chatpage.test.js

```diff
@@ -203,13 +208,13 @@ describe('<ChatPage />', () => {
 			expect(RETURN_VALUE).toBeUndefined()
 		})
 		it('Should handle the join event and update state', () => {
-			const JOIN_DATA = {roomId: DUMMY_ROOM, usernames: DUMMY_USER}
+			const JOIN_DATA = {roomId: DUMMY_ROOM, username: DUMMY_USER}
 			const RETURN_VALUE = suite.Component.instance().handleJoinEvent(JOIN_DATA)
 			const USERROOM_ROOMID = suite.Component.state(USERROOMS)[FIRST_INDEX].roomId
 			const USERROOM_USERNAMES = suite.Component.state(USERROOMS)[FIRST_INDEX].usernames
```

Comment:
Do we want to use `usernames` here, or `username`?

Reply:
These tests will be updated when I add functionality for multiple users of a single room.